torch/nn/functional.py

```diff
@@ -3836,6 +3853,14 @@ def interpolate(input: Tensor, size: Optional[int] = None, scale_factor: Optiona
         backward compatibility.
         Mode ``mode='nearest'`` matches buggy OpenCV's ``INTER_NEAREST`` interpolation algorithm.
 
+    .. note::
+        ``round_with_scale_factor`` argument fixes the output size issue when ``scale_factor`` is
+        provided. Historically, the output size is computed as ``output_size = int(input_size * scale_factor)`
+        which does not match with OpenCV, Scikit-Image, Scipy. Correct way to compute the output size is to
+        ``output_size = round(input_size * scale_factor)``. The difference of +1 in the output size is
+        visible for floating ``scale_factor`` values.
+        Related issue: https://github.com/pytorch/pytorch/issues/62396
```

Comment:
nit: make related issue a link that points to the issue rather than pasting the url